cmd/plume/prune.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2020 Kinvolk GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+
+	"github.com/coreos/mantle/platform/api/aws"
+	"github.com/coreos/mantle/platform/api/azure"
+)
+
+var (
+	days        int
+	pruneDryRun bool
+	cmdPrune    = &cobra.Command{
+		Use:   "prune [options]",
+		Short: "Prune old release images.",
```

Comment:
and maybe document that it takes `--channel` into account

Reply:
Yeah, I mentioned that in the how to test command (I was using `--version none`), but I figured that instead of documenting this brokenness, it's better to just override it. I've added a line to always set the version to none.

Reply:
I've documented the fact that it requires the channel now.